src/main/resources/studentEmailFragment-courseJoin.html

```diff
@@ -18,4 +18,8 @@
     However, we recommend joining the course using your Google account,
     because it gives you more convenient access to all your feedback
     stored in TEAMMATES.
+</p>
+
+<p>
+    For any non-technical queries (eg: Assigned to the wrong team, misspelled name, etc.), you can contact your instructor(s) at: ${coOwnersEmails}
```

Comment:
`(e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.)`